shell/platform/windows/flutter_windows_view.cc

```diff
@@ -56,6 +56,7 @@ void FlutterWindowsView::SetEngine(
   engine_ = std::move(engine);
 
   engine_->SetView(this);
+  binding_handler_->AfterEngineSet();
```

Comment:
Can we make this work the same way the window's width and height are sent to the engine?

1. At startup, this method calls `FlutterWindow::GetPhysicalWindowBounds` and sends bounds down to the engine through `SendWindowMetrics`
2. Whenever the window is resized, `FlutterWindow::OnResize` calls `FlutterWindowsView::OnWindowSizeChanged`, which sends bounds down to the engine using `SendWindowMetrics`

Reply:
See if newly pushed changes are better

Reply:
Looks good! Resolving